Curve-finance-suite/Registry-Remove-Pool/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "curve-13",
```

Comment:
`name` can't be shared between agents, so this agent should be mixed with `PoolAdded`